admin/client/components/Forms/EditForm.js

```diff
@@ -98,6 +101,73 @@ var EditForm = React.createClass({
 			confirmationDialog: null,
 		});
 	},
+	updateItem () {
+		const { data, list } = this.props;
+		const editForm = this.refs.editForm.getDOMNode();
+		const formData = new FormData(editForm);
+
+		// Clear alerts
+		this.setState({
+			err: null,
+			success: null,
+			loading: true,
+		});
+
+		list.updateItem(data.id, formData, (err, data) => {
+			// TODO: implement smooth scolling
+			scrollTo(0, 0); // Scroll to top
+			if (err) {
+				this.setState({
+					err: err,
+					loading: false,
+				});
+			} else {
+				// Success, display success flash messages, replace values
+				// TODO: Update key value
```

Comment:
Is it a key based on a field value that may get updated but the updated key value is not transferred to the UI via `values: data.fields,`